packages/usdk/cli.js

```diff
@@ -106,6 +109,8 @@ import { getLatestVersion } from './lib/version.mjs';
 //   getDirectoryHash,
 // } from './util/hash-util.mjs';
 
+globalThis.WebSocket = WebSocket; // polyfill for multiplayer library
```

Comment:
Is this needed? I don't understand why the browser would need an additional `WebSocket`.

Reply:
The only case I saw that this was still needed was `electron`.